CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Performance enhancements
+* `skbio.tree.nj` wall-clock time was decreased about 66% by vectorizing `skbio.tree._nj._compute_q`.
```

Comment:
Sorry for not specifying this before, but could you please add a reference to this PR? eg 

```text
* `skbio.tree.nj` wall-clock time was decreased about 66% by vectorizing `skbio.tree._nj._compute_q`. ([#1512](https://github.com/biocore/scikit-bio/pull/1512))
```